### ISSUE TITLE
Use GitHub action to raise Antora UI version upgrade issue

### ISSUE THREAD

**Issue by philwebb** [spring-projects/spring-boot#40928]
https://github.com/spring-io/spring-doc-actions?tab=readme-ov-file#update-antora-ui-spring

**Comment by wilkinsona** [comment]
I'm not sure this is really worth it. AIUI, the action will create a PR which we'd then have to merge. It'd be just as easy to watch [the repo](https://github.com/spring-io/antora-ui-spring) for releases and then do the upgrade manually. It's only a one-liner after all.

**Comment by philwebb** [comment]
Yeah, I agree. I wonder if we can rip out the logic to tell if there's an update and raise an issue so we don't need to manually watch the repo.

**Comment by wilkinsona** [comment]
Given how infrequently new versions of the UI are released, I'm going to close this one. Having an action that runs periodically to check for a new version feels like a bit of a waste of resources when there's almost a year between 0.4.18 and 0.4.19 (the latest). We can reconsider if the releases become more frequent and we realise we've missed one.
